docs/extensions/OnDeviceMarkup.md

```diff
@@ -0,0 +1,47 @@
+---
+title: OnDevice Markup Extension
+author: sonnemaf
+description: The OnDevice markup extension allows you to customize UI appearance on a per-DeviceFamily basis.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, device family, markup extension, XAML, markup 
+---
+
+# OnDevice Markup Extension
+The [OnDevice Markup Extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.ondevice) allows you to customize UI appearance on a per-DeviceFamily basis. It is inspired on the OnPlatform(https://github.com/xamarin/Xamarin.Forms/issues/2608) markup extensions from Xamarin.Forms 3.2
```

Comment:
@sonnemaf sorry, just noticed looking at the final preview, you forgot the `[ ]` around the 'OnPlatform' text to make it a clean-looking link. :)

Reply:
```suggestion
The [OnDevice Markup Extension](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.uwp.ui.extensions.ondevice) allows you to customize UI appearance on a per-DeviceFamily basis. It is inspired on the [OnPlatform](https://github.com/xamarin/Xamarin.Forms/issues/2608) markup extensions from Xamarin.Forms 3.2
```

@sonnemaf can you accept this one small fix so we can merge this?  Thanks!